ikarus/python/test/debug_info.py

```diff
@@ -1,15 +1,24 @@
 # SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
 # SPDX-License-Identifier: LGPL-3.0-or-later
 import os
+import dune.generator as generator
+
 
 os.environ["DUNE_LOG_LEVEL"] = "debug"
 os.environ["DUNE_SAVE_BUILD"] = "terminal"
 
 
 def setDebugFlags():
-    import dune.generator as generator
+    def apply_debug_flags():
+        generator.setFlags("-g ", noChecks=False)
+
+    # Check if the environment variable is set
+    build_type = os.environ.get("IKARUS_PYTHON_TEST_BUILD_TYPE_OVERRIDE")
 
-    generator.setFlags("-g ", noChecks=False)
+    # Apply flags only if the build type is Debug, or if the variable is not set
+    if build_type == "Debug" or build_type is None:
+        apply_debug_flags()
+        print("Python Generator: Debug Flag enabled")
```

Comment:
Ah thats good to see if it is really set. Maybe to have the same line for release to be super sure when someone debugs this, we can just write this out for both version. Like:

print("JIT Python Bindings BUILD_TYPE: Debug")
print("JIT Python Bindings BUILD_TYPE: Release")